doc/explanations/faq.rst

```diff
@@ -1,218 +1,337 @@
+transactions. These transactions must also carry the minting policies
+for the tokens they are minting, so that they can be checked during
+validation.
+
+The result of processing a minting transaction is that the ledger will
+now additionally contain the assets included in the mint field (minting
+field) of the transaction. If the quantity of a particular asset in the
+mint field is negative, the result is that after processing the
+transaction, the total quantity of that specific asset on the ledger
+will be reduced by the amount reflected in the mint field.
+
+Note that a single transaction might mint tokens associated with
+multiple distinct minting policies. e.g.,
+``(Policy1, SomeTokens), (Policy2, SomeOtherTokens)``. Note also that a
+transaction might simultaneously mint some tokens and burn some other
+ones.
+
+**Q. What is a minting policy?**
+
+A. A minting policy is a set of rules used to regulate the minting of
+assets associated with it (scoped under it). For example, who has
+control (and under what conditions) over the supply of the currency, and
+its minting and burning. These rules are about the content of the
+transaction data of the transaction that is attempting the mint. e.g., a
+minting policy can require a particular set of keys to have signed the
+minting transaction.
+
+This set of rules is defined by the user who wishes to create the new
+asset. For example, a user might wish to only allow themselves to ever
+mint this particular kind of token. In this case, they would stipulate
+this in the policy. The node checks adherence to minting policies when a
+transaction is processed by running the code or checking the relevant
+signatures. Transaction data must satisfy all the minting policies of
+all assets the transaction is attempting to mint.
+
+**Q. What is a token builder and what is its functionality?**
+
+A. A token builder is a piece of software that allows the user to define
+the tokens to be minted and include them in a minting transaction. It
+also ensures that the appropriate additional data needed to verify that
+the transaction is allowed to perform the mint is included in the
+transaction (see minting policy question below).
 
 Policy examples and ways to define policies
-################################################
+===========================================
 
-What is 'multisig' and how is it related to minting policies?
-********************************************************************************
+**Q. What is ‘multisig’ and how is it related to minting policies?**
 
-The multisig scripting language (which existed prior to the introduction of MA functionality on Cardano) specifies some minimal set of signatures required to allow a transaction to perform a certain action, usually to spend a UTxO entry.
+A. The multisig scripting language (which existed prior to the
+introduction of MA functionality on Cardano) specifies some minimal set
```

Comment:
the multi-sig DSL was actually introduced in the Shelley ledger era.

Reply:
Tweaked to avoid inaccuracy